### PR TITLE
[feature] glue Add to CommandFactory

### PR DESCRIPTION
**Pull request by Sungju-Huh**
including <sstream> to avoid link error.

Signed-off-by: Sungju Huh <sjslush@gmail.com>

### CODE REVIEW

File: EmployeeManagement/EmployeeManagement/CommandFactory.cpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 shared_ptr<ICommand> CommandFactory::createCommand(Input input) {
 	if (input.getCommand() == "ADD") {
-		shared_ptr<ICommand> result = nullptr; // new AddCommand(input)
+		shared_ptr<ICommand> result = make_shared<Add>(input);
```
Comment (yh0203kim) [inline EmployeeManagement/EmployeeManagement/CommandFactory.cpp:5]: CommandFactory에 Add 연결함

**Review by yh0203kim — DISMISSED** [review]
LGTM:)

File: EmployeeManagement/EmployeeManagement/CommandFactory.cpp [hunk 2]
```diff
@@ -14,7 +14,7 @@ shared_ptr<ICommand> CommandFactory::createCommand(Input input) {
 		return result;
 	}
 	else if (input.getCommand() == "SCH") {
-		shared_ptr<ICommand> result = nullptr; // new SchCommand(input)
+		shared_ptr<ICommand> result = make_shared<Sch>(input);
```
Comment (yh0203kim) [inline EmployeeManagement/EmployeeManagement/CommandFactory.cpp:17]: CommandFactory에 Sch 연결함

**Review by h2kfl — APPROVED** [review]

**Review by yh0203kim — APPROVED** [review]
LGTM!

**Pull request merged**